sycl/include/CL/sycl/handler.hpp

```diff
@@ -107,7 +107,31 @@ template <typename Type> struct get_kernel_name_t<detail::auto_name, Type> {
 
 __SYCL_EXPORT device getDeviceFromHandler(handler &);
 
-device getDeviceFromHandler(handler &);
+/// These are the forward declaration for the classes that help to create
```

Comment:
Please remove this section of code, it is unrelated.

Reply:
OK, wrong merge I assume